### PR TITLE
Add hidapi recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@prusnak do you agree with being added as a maintainer?

**Comment by prusnak** [comment]
> @prusnak do you agree with being added as a maintainer?

No. I don't plan to maintain it here.

**Comment by goanpeca** [comment]
Thanks for the reply @prusnak

File: recipes/hidapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/trezor/cython-hidapi
+  summary: A Cython interface to the hidapi from https://github.com/libusb/hidapi
+  license: Unlicense
```
Comment (jaimergp) [inline recipes/hidapi/meta.yaml:44]:
```suggestion
  license: Unlicense OR GPL-3.0-only OR BSD-3-Clause
```

File: recipes/hidapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  license_file: LICENSE-orig.txt
```
Comment (jaimergp) [inline recipes/hidapi/meta.yaml:45]:
```suggestion
  license_file: 
    - LICENSE-orig.txt
    - LICENSE-bsd.txt
    - LICENSE-gpl3.txt
    - libusb_hidapi_licenses.txt
```

**Pull request merged**